lib/trailblazer/operation/wrap.rb

```diff
@@ -1,7 +1,8 @@
 class Trailblazer::Operation
-  def self.Wrap(user_wrap, id: "Wrap/#{rand(100)}", &block)
+  def self.Wrap(user_wrap, id: nil, &block)
     operation_class = Wrap.create_operation(block)
     wrapped         = Wrap::Wrapped.new(operation_class, user_wrap)
+    id            ||= "Wrap/#{user_wrap}"
```

Comment:
You can write that like `self.Wrap(user_wrap, id: "Wrap/#{user_wrap}", &block)`.

This will break if you wrap the same, say, `MyTransaction` twice, though.